### PR TITLE
GAP and GMP in caps / Dist script for website

### PR DESCRIPTION
**Pull request by sebasguts**

### CODE REVIEW

**Comment by fingolfin** [comment]
Sorry, I also changed the GAP / GMP spelling, so there is now a conflict. Can you fix this PR? (juse use `git rebase -i` and drop the first commit; and perhaps squash some of the other commits ;-).

**Comment by fingolfin** [comment]
What is the point of the ` delete_old_releases.py` script? Why would we want to delete old releases? I'd normally think it's a good idea to keep old releases around.

File: make_dist_for_normaliz.sh [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+cd NormalizInterface
+gap -A -q -b <<EOF
+# HACK
+MakeReadWriteGlobal("SetPackageInfo");
+SetPackageInfo:=function(pkg)
+  PrintTo("VERSION",pkg.Version);
+end;;
+Read("PackageInfo.g");
+QUIT;
+EOF
```
Comment (fingolfin) [inline make_dist_for_normaliz.sh]: So why is this HACK here? It seems to be a workaround for somebody forgetting to update the VERSION file in the repository, but then that's a bug by itself already....

Comment (sebasguts) [inline make_dist_for_normaliz.sh]: Is the VERSION file really needed? As you see, the next lines delete the new file anyway.

**Comment by sebasguts** [comment]
I will do the rebase right now. GitHub limites the space for binary files, so I decided on keeping only three releases. Should we have more?

**Comment by fingolfin** [comment]
> GitHub limites the space for binary files
Source for this claim? The only limitation I am aware of is that individual files must be less than 2 GB.

**Comment by csoeger** [comment]
On file sizes I found:
> GitHub will warn you when pushing files larger than 50 MB. You will not be allowed to push files larger than 100 MB.
> If your repository exceeds 1GB, you might receive a polite email from GitHub Support requesting that you reduce the size of the repository to bring it back down."

File: make_dist_for_normaliz.sh (hunk 1 above)
Comment (fingolfin) [inline make_dist_for_normaliz.sh]: The VERSIOn file is need for the configure script.

**Comment by fingolfin** [comment]
@csoeger Yeah but that is only for repository content. For releases, the relevant page is https://help.github.com/articles/about-releases/ and it only mentions the 2GB-per-file limit.

File: make_dist_for_normaliz.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+rm json_data
+
+cd gh-pages
+git add *tar.gz
+cp -f ../PackageInfo.g README.md .
+
+cp -f ../doc/*.{css,html,js,txt} doc/
+gap update.g
+git add PackageInfo.g README.md README_layouted.md doc/ _data/package.yml
```
Comment (csoeger) [inline make_dist_for_normaliz.sh]: Remove `README_layouted.md` also here?

**Pull request closed without merging**